depthai_nodes/ml/parsers/yolo.py

```diff
@@ -66,6 +79,9 @@ def __init__(
         self.iou_threshold = iou_threshold
         self.mask_conf = mask_conf
         self.n_keypoints = n_keypoints
+        self.anchors = anchors if anchors is not None else [None, None, None]
```

Comment:
```suggestion
        self.anchors = anchors or [None, None, None]
```
Should be the same AFAIK

Reply:
Ok. Adding both changes.